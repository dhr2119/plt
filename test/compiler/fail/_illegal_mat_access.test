int main() {
  int x;
  x = 5;
  x = x[0];
  return 0;
}
