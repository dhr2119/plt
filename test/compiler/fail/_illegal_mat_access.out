Fatal error: exception Exceptions.VectorAccessOnNonMatrix("x")
